docs/source/Security_Services_Architectural_Maturity_Index/SSAM_Rubric.md

```diff
@@ -1,30 +1,30 @@
 
 # Rubric
 
-## Reliability
+## Reliability and Quality
 
 The Component (or the substantial logic thereof) is provided by a
 reputable party and actively maintained.
 
-  - 0 – Component is unvetted
+  - 0 – Written in-house with minimal documentation or third-party component that is uncommon and/or not actively supported
 
-  - 1 – Vetted component is used, but is not a current version
+  - 1 – Vetted component is used, but is not a current versionThird-party component is used, but may not be a current version or actively supported 
 
-  - 2 – Mature, vetted component with multiple active contributors
+  - 2 – Mature, vetted component with multiple active contributorsMature, third-party component with multiple active contributors; configured by secure best practices/guidelines
```

Comment:
Looks like there's a punctuation and space missing here.

Reply:
Fixed